PARTICIPANTES.md

```diff
@@ -71,4 +71,8 @@ GitHub: https://github.com/samuelramox
 
 - Victor Eduardo Santos Soares  
 Email: v_eduardo@live.com  
-GitHub: https://github.com/veduardo93  
\ No newline at end of file
+GitHub: https://github.com/veduardo93  
+
+- Allan Lima
```

Comment:
Importante: Leia as instruções no início da página!

1) As linhas com o sue nome, github e email *tem* que ser terminadas com 2 espaços, ou o markdown vai formatar tudo como um parágrafo só.

2) Ordem alfabética por sobrenome.